<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contact</class>
 <widget class="QWidget" name="Contact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(206, 226, 255);
}
</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>80</y>
     <width>531</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
	background-color: rgb(234, 246, 255);
	border-radius:4px;
}
QListWidget::Item:hover{
	background: rgb(234, 246, 255); 
	
	selection-background-color: rgb(234, 246, 255);
	gridline-color: (234, 246, 255);
	selection-color: (234, 246, 255);
	border-width: 0px;
}
QListWidget::Item:selected{
	background: rgb(234, 246, 255); 
	
	selection-background-color: rgb(234, 246, 255);
	gridline-color: (234, 246, 255);
	selection-color: (234, 246, 255);
	gridline-color: rgb(234, 246, 255);
	border-width: 0px;
	selection-color: rgb(234, 246, 255);
}
QListWidget::Item:selected:!active{
	background: rgb(234, 246, 255); 
	
	selection-background-color: rgb(234, 246, 255);
	gridline-color: (0,0,0);
	selection-color: (234, 246, 255);
	gridline-color: rgb(234, 246, 255);
	border-width: 0px;
	selection-color: rgb(234, 246, 255);
}</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>417</y>
     <width>531</width>
     <height>123</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	background-color: rgb(234, 246, 255);
	border-width:0px;
	border-radius:4px;	
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="headimage">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>19</y>
     <width>55</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sign">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>171</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	background-image: url(:/Resources/LoginWindow/qqshow.png);
	border-bottom-left-radius: 3px; 
	border-bottom-right-radius: 3px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>451</y>
     <width>173</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-image: url(:/Resources/LoginWindow/showroot.png);
border-radius:0px;
	border-bottom-left-radius: 0px; 
	border-bottom-right-radius: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>513</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed{
	
	background-color: rgb(6, 118, 255);
}
QPushButton:hover{
	
	background-color: rgb(7, 181, 255);
}
QPushButton{
	
	background-color: rgb(30, 195, 255);
	border-radius:4px;
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="land">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>394</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}
QPushButton:hover{
	border-radius:2px;
	
	
	color: rgb(185, 215, 253);
	background-color: rgb(185, 215, 253);
	
}
QPushButton:press{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qt-myqq.qrc">
        <normaloff>:/Resources/Icon/font-symbol-of-letter-a.svg</normaloff>:/Resources/Icon/font-symbol-of-letter-a.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}
QPushButton:hover{
	border-radius:2px;
	
	
	color: rgb(185, 215, 253);
	background-color: rgb(185, 215, 253);
	
}
QPushButton:press{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qt-myqq.qrc">
        <normaloff>:/Resources/Icon/smile.svg</normaloff>:/Resources/Icon/smile.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}
QPushButton:hover{
	border-radius:2px;
	
	
	color: rgb(185, 215, 253);
	background-color: rgb(185, 215, 253);
	
}
QPushButton:press{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qt-myqq.qrc">
        <normaloff>:/Resources/Icon/screenshot.svg</normaloff>:/Resources/Icon/screenshot.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}
QPushButton:hover{
	border-radius:2px;
	
	
	color: rgb(185, 215, 253);
	background-color: rgb(185, 215, 253);
	
}
QPushButton:press{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qt-myqq.qrc">
        <normaloff>:/Resources/Icon/microphone.svg</normaloff>:/Resources/Icon/microphone.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}
QPushButton:hover{
	border-radius:2px;
	
	
	color: rgb(185, 215, 253);
	background-color: rgb(185, 215, 253);
	
}
QPushButton:press{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qt-myqq.qrc">
        <normaloff>:/Resources/Icon/picture.svg</normaloff>:/Resources/Icon/picture.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fileButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}
QPushButton:hover{
	border-radius:2px;
	
	
	color: rgb(185, 215, 253);
	background-color: rgb(185, 215, 253);
	
}
QPushButton:press{
	border-radius:2px;
	border-color: rgb(206, 226, 255);
	background-color: rgb(206, 226, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qt-myqq.qrc">
        <normaloff>:/Resources/Icon/folder (1).svg</normaloff>:/Resources/Icon/folder (1).svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>land</zorder>
  <zorder>listWidget</zorder>
  <zorder>textEdit</zorder>
  <zorder>headimage</zorder>
  <zorder>sign</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources>
  <include location="qt-myqq.qrc"/>
 </resources>
 <connections/>
</ui>
